discord-bot/README.md

```diff
@@ -51,6 +51,20 @@ Remember to save your changes.
 https://discord.com/oauth2/authorize?client_id=YOUR_CLIENT_ID_HERE&permissions=8&scope=bot%20applications.commands
 ```
 
+## Discord setup for development
```

Comment:
is this a different workflow from the "Bot Setup" above? could the two be merged or do they refer to separate applications?

Reply:
This is for the `website` part to enable discord authentication

Reply:
When we were discussing it in the issue, I was told to put it in `discord-bot` so, I put it there. I'll just change it